pitest-maven/src/test/java/org/pitest/maven/MojoToReportOptionsConverterTest.java

```diff
@@ -376,7 +377,7 @@ public void testParsesExcludedClasspathElements()
   public void testParsesSurefireConfigWhenFlagSet() {
     parseConfig("<parseSurefireConfig>true</parseSurefireConfig>");
     verify(this.surefireConverter).update(any(ReportOptions.class),
-        any(Xpp3Dom.class));
+        isNull());
```

Comment:
From looking at the code of MojoToReportOptionsConverter and maven classes I doubt this will ever be null. But mocking this out will be cumbersome. Maybe that's not worth the effort.

Reply:
The reason of `null `is that `surifire` (MojoToReportOptionsConverterTest:54) has no configuration